.github/unfinished-workflows/linux_test.yml

```diff
@@ -0,0 +1,43 @@
+name: Linux Test - Release Mode
```

Comment:
Anything in the unfinished-workflows folder should be considered just a work in progress and not ready to go.  But would be nice to get them finished sometime.